JamesNojekChallenge/Database.py

```diff
@@ -0,0 +1,43 @@
+import json
+
+def oldToNewDB(filename):
+    try:   
+        with open(filename, "r") as f:
+            text = json.load(f) #get data from file
+            print("Input the new database name: ")
+            DBname = str(input())   #select database to edit

```

Comment:
How does the code know how and when to connect and disconnect from the actual database?

Reply:
See my other comment about SQL inexperience; the code doesn't know because I didn't know that was necessary.